src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -0,0 +1,17 @@
+import { Avatar } from "@mui/material";
+import styles from "./UserAvatar.module.scss";
+
+export * from "./UserAvatar";
+
+type userAvatarProps = {
+  label: string;
+  imageSrc?: string;
+};
+
+export const UserAvatar = ({ label, imageSrc }: userAvatarProps) => {
+  return (
+    <Avatar className={styles.avatarSize} alt={label} src={imageSrc}>
```

Comment:
A może dodać mu jakiś lepszy opis?
```suggestion
export const UserAvatar = ({ label, imageSrc }: userAvatarProps) => {
const imageDescription = + "user avatar of " +label 
  return (
    <Avatar className={styles.avatarSize} alt={imageDescription} src={imageSrc}>
```

Reply:
fajny pomysł :) Zrobione